Tests/MapboxNavigationTests/RouteControllerSnapshotTests.swift

```diff
@@ -1,18 +1,15 @@
 import XCTest
-import FBSnapshotTestCase
 import Turf
 import MapboxDirections
 import TestHelper
 @testable import MapboxCoreNavigation
 @testable import MapboxNavigation
```

Comment:
As far as I can tell, this file no longer uses MapboxNavigation, so we can remove this import statement. The test suite exercises RouteController, a class vended by MapboxCoreNavigation, so we should move this file to the MapboxCoreNavigationTests target.

Reply:
Thanks for the suggestion, I'll change the target of this file.